help/en/releasenotes/current-draft-note.shtml

```diff
@@ -118,8 +118,13 @@
             <ul>
                 <li>NCE serial connections have been updated to use a new serial library.</li>
                 <li>Clock, Macro, and Consist added and improved error notification to user
-                	when invalid inputs entered. Prior these were silently ignored or only
-                	logged to the console.</li>
+               	when invalid inputs entered. Prior these were silently ignored or only
+               	logged to the console.</li>
+               	<li>NCE Packet Analyzer has internal improvements to prevent errors due to port not
+               	being open. Duplicates are now given as a count instead of a row of +++ as
+               	the time stamp wasn't being placed correctly and a count is more meaningful.
+               	The option to close the connection was added to manage selecting a different
+               	port. It now supports the old and new versions of the analyzer.</li>
```

Comment:
Indentation seems to be wrong here. Maybe TABs vs spaces?

It doesn't matter for the users, but might be confusing for developers who want to add anything here.

Reply:
Or it might be that you forgot that the text after `<li>` was intended and that you didn't do that for your `<li>`.